### PR TITLE
feat: add user guide HP-2322

### PR DESCRIPTION
**Pull request by Riippi**
Add pages and downloads for user guide

https://helsinkisolutionoffice.atlassian.net/browse/HP-2322

Updated translations so also this ticket is covered
https://helsinkisolutionoffice.atlassian.net/browse/HP-2465

### CODE REVIEW

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

File: src/common/footer/Footer.tsx [hunk 1]
```diff
@@ -79,6 +79,11 @@ const Footer = () => {
           label={t('cookies.pageName')}
         />
         <HDSFooter.Link as={Link} to="/about" label={t('footer.about')} />
+        <HDSFooter.Link
+          as={Link}
+          to="/guide"
+          label={'Helsinki-profiilin käyttöohje'}
```
Comment (mikkojamG) [inline src/common/footer/Footer.tsx]: Probably needs translation? 🤔

File: src/profile/components/tableOfContents/TableOfContents.module.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
+.table-of-contents {
+    max-width: 1296px;
+    margin-top: 32px;
```
Comment (mikkojamG) [inline src/profile/components/tableOfContents/TableOfContents.module.css]: It's a good practice to use HDS variables when possible, for instance spacing of 32px could use `--spacing-layout-s` or `--spacing-l`, both are equivalent of `2rem` 🤔

File: src/profile/components/tableOfContents/TableOfContents.module.css [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+@media(min-width: 992px) {
+    .table-of-contents+* {
+        margin-top:64px
+    }
+}
+
+.table-of-contents__container {
+    background-color: #f7f7f8;
```
Comment (mikkojamG) [inline src/profile/components/tableOfContents/TableOfContents.module.css]: Could replace with `--color-silver-light`

File: src/profile/components/tableOfContents/TableOfContents.module.css [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    border-left: 8px solid #0000BF;
```
Comment (mikkojamG) [inline src/profile/components/tableOfContents/TableOfContents.module.css]: Could replace with `--color-bus`

File: src/profile/components/tableOfContents/TableOfContents.module.css [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    max-width: 908px;
+    padding: 32px 24px
+}
+
+.table-of-contents__title {
+    --line-height: 1.3333333333;
```
Comment (mikkojamG) [inline src/profile/components/tableOfContents/TableOfContents.module.css]: Would `--lineheight-m` or `--lineheight-l` be close enough? 🤔

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

File: src/userGuide/UserGuideImage.tsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+interface ImageProps {
+  alt: string;
+  src: string;
+}
+
+function UserGuideImage({ src, alt }: ImageProps): React.ReactElement {
+  return (
+    <p>
+      <img src={src} alt={alt} />
```
Comment (mikkojamG) [inline src/userGuide/UserGuideImage.tsx]: Could try adding `loading="lazy"`, could be nice since there are so many pictures to be loaded 🤔

**Review by mikkojamG — CHANGES_REQUESTED** [review]

File: src/userGuide/UserGuideAccordion.tsx [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+  const theme = {
+    '--background-color': '#F2F2F2',
+  };
+
+  const accordionProps: AccordionProps = {
+    style: accordionStyle,
+    size: isMobile ? 's' : 'l',
+    card: true,
+    language: 'en',
```
Comment (arpargo) [inline src/userGuide/UserGuideAccordion.tsx]: Language should be parametrized with current language to have the accordion close buttons translated.

**Review by arpargo — CHANGES_REQUESTED** [review]

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

File: src/userGuide/UserGuideAccordion.tsx (hunk 7 above)
Comment (Riippi) [reply src/userGuide/UserGuideAccordion.tsx]: Now it's passed

File: src/userGuide/UserGuideImage.tsx (hunk 6 above)
Comment (Riippi) [reply src/userGuide/UserGuideImage.tsx]: Everything is lazy now

File: src/profile/components/tableOfContents/TableOfContents.module.css (hunk 5 above)
Comment (Riippi) [reply src/profile/components/tableOfContents/TableOfContents.module.css]: fixed

File: src/profile/components/tableOfContents/TableOfContents.module.css (hunk 4 above)
Comment (Riippi) [reply src/profile/components/tableOfContents/TableOfContents.module.css]: fixed

File: src/profile/components/tableOfContents/TableOfContents.module.css (hunk 3 above)
Comment (Riippi) [reply src/profile/components/tableOfContents/TableOfContents.module.css]: fixed

File: src/profile/components/tableOfContents/TableOfContents.module.css (hunk 2 above)
Comment (Riippi) [reply src/profile/components/tableOfContents/TableOfContents.module.css]: fixed

File: src/common/footer/Footer.tsx (hunk 1 above)
Comment (Riippi) [reply src/common/footer/Footer.tsx]: Translations added

**Review by mikkojamG — APPROVED** [review]

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr352-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Review by arpargo — APPROVED** [review]
LGTM

**Pull request merged**